homeassistant/components/webhook.py

```diff
@@ -85,7 +85,7 @@ class WebhookView(HomeAssistantView):
             return Response(status=200)
 
         try:
-            response = await handler(hass, webhook_id, data)
```

Comment:
I suggest that we drop `data` and just pass `request`

Reply:
OK. I'll clean up the rest of webhook and make the necessary changes in ifttt and update the tests (if needed)

Reply:
I just looked at the code. If we drop data altogether, each component will have to implement the JSON validation on its own. Upside is that the webhook component can be used for calls that submit non-JSON data.

Reply:
Yeah I think that is fine . Gives most flexibility.